test/unit/alignment/pairwise/align_pairwise_test.cpp

```diff
@@ -49,9 +50,9 @@ auto call_alignment(seq_t && seq, cfg_t && cfg)
     {
         return seqan3::align_pairwise(std::forward<seq_t>(seq), std::forward<cfg_t>(cfg));
     }
-    else
+    else if constexpr (std::same_as<type_param_t, seqan3::align_cfg::parallel>)
     {
-        auto && config = cfg | type_param_t{};
+        auto && config = cfg | seqan3::align_cfg::parallel{4};
```

Comment:
I don't understand this change?

Reply:
`type_param_t` is either void or `seqan3::align_cfg::parallel`, because we want to test the serial and the parallel version. And if it is parallel, we do not want to use the default initialisation, because half of the tests would fail with the std::bad_optional_access exception. So I initialise the parallel tests with 4.

Reply:
thank you for the explanation :)